main.go

```diff
@@ -193,13 +194,18 @@ func buildConnStr(resource string) string {
 // Pull API Specs from configuration
 var APILoader = APIDefinitionLoader{}
 
-func getAPISpecs() []*APISpec {
-	var apiSpecs []*APISpec
+// Currently loaded specs
+var APISpecs []*APISpec
```

Comment:
Also, if you undo the unexport the diff will get smaller as you don't have to modify all the names below